install.ps1

```diff
@@ -32,7 +35,10 @@ function Help
     Write-Host "Flags:"
     Write-Host "  -VPN                  Enables support for enterprise VPNs."
     Write-Host "  -WindowsContainers    Enables support for Windows Containers using Docker binary."
-    Write-Host "  -Alias                Creates alias for usual Docker commands in Powershell."
+    Write-Host "  -Alias                Creates alias for usual Docker commands in Powershell and Bash."
+    Write-Host ""
+    Write-Host "Advanced Flags:"
+    Write-Host "  -RenameBinaries               Renames binaries to provide universal docker command support in cases where shell profiles are of no use, but comes with some caveats (e.g. requires using docker compose instead of docker-compose). Incompatible with -Alias flag."
```

Comment:
```suggestion
    Write-Host "  -RenameBinaries       Renames binaries to provide universal docker command support in cases where shell profiles are of no use, but comes with some caveats (e.g. requires using docker compose instead of docker-compose). Incompatible with -Alias flag."
```